appservice/src/intTest/groovy/nu/yona/server/UpdateBuddyUserInfoTest.groovy

```diff
@@ -40,6 +40,7 @@ class UpdateBuddyUserInfoTest extends AbstractAppServiceIntegrationTest
 
 		User richardAfterProcess = appService.reloadUser(richard)
 		richardAfterProcess.buddies[0].nickname == "Bobby"
```

Comment:
Please create a task to remove the nickname from the buddy DTO after the app has been updated.

Reply:
I have a note on my desk to do that, including the embedded goals

Reply:
Done: [YD-505](http://jira.yona.nu/browse/YD-505).